### DISCUSSION TITLE
SyntaxError: Expected "%", "{", or end of input but "|" found.

### DISCUSSION THREAD

**Discussion by t1m013y** [gildas-lormeau/SingleFile#1312] [General]
**Describe the bug**
Error while saving any page: `SingleFile error: SyntaxError: Expected "%", "{", or end of input but "|" found.`

**To Reproduce**
1. Go to any website
2. Click on save page
3. See error at top of the page

**Expected behavior**
Successful page saving or showing annotation editor

**Screenshots**
![изображение](https://github.com/gildas-lormeau/SingleFile/assets/91249047/21e6bb46-cdf2-4847-a281-6245141ba6aa)

**Environment**
 - OS: Windows 10
 - Browser: Mozilla Firefox
 - Version:  119.0.1 (64-разрядный)

**Additional context**
Tried to save multiple pages: wikipedia, google etc but no one saved correctly.

**Comment by t1m013y** [comment]
Found reason of the error: default info panel content was set to `{page-title} | {url-href} | {date-locale} {time-locale}`. I replaced it with `{page-title} \| {url-href} \| {date-locale} {time-locale}`

**Comment by gildas-lormeau** [comment]
Thank you for the report. I confirm that "|" is special character since the support of function calls have been added in templates. I did not anticipate people would use this special character for the infobar, unfortunately. That's why it caused a regression for you. Right now, I won't implement a workaround for this because it would be a bit convoluted. However, I'm converting this report into a "discussion" to make it more visible.
